static/js/components/ProfileImageUploader.js

```diff
@@ -74,9 +77,9 @@ const ProfileImageUploader = ({
       </Button>,
       <Button
         type='button'
-        className={`save-button ${photo ? 'primary-button' : 'secondary-button disabled'}`}
+        className={`save-button ${!disabled ? 'primary-button' : 'secondary-button disabled'}`}
         key="save"
-        onClick={photo ? updateUserPhoto : undefined}>
+        onClick={!disabled ? updateUserPhoto : undefined}>
```

Comment:
could you change these ternarys to just check `disabled` (and therefore be reversed)? I think it might be a little clearer.